components/BackToTop/BackToTopButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import { FaArrowUp } from "react-icons/fa";
+import { SCROLL_LIMIT } from "../../app/constants";
+
+export const BackToTopButton = () => {
+  const [scrollY, setScrollY] = useState(0);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      setScrollY(window.scrollY);
+    };
+
+    handleScroll();
+
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+
+  }, []);
+
+  const handleClick = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    })
+  }
+
+  if (scrollY > SCROLL_LIMIT) {
+    return (
+      <div className="group fixed bottom-12 right-12">
+        <button 
+          className="focus:animate-button-press rounded-2xl border border-gray-100 bg-violet-600 p-4 text-white shadow-xl focus:ring group-hover:border-dashed group-hover:border-violet-400 group-hover:bg-white dark:drop-shadow-[5px_5px_8px_rgba(124,58,237,0.25)] dark:group-hover:bg-[#101623]"
+          onClick={handleClick}>
+          <FaArrowUp className="group-hover:text-violet-500" />
+        </button>
+        <span className="absolute left-1/2 top-1/2 -z-10 -translate-x-1/2 -translate-y-1/2 rotate-0 text-5xl transition-all duration-100 ease-in-out group-hover:-ml-8 group-hover:-rotate-45">
```

Comment:
```suggestion
        <span className="absolute left-1/2 top-1/2 -z-10 -translate-x-1/2 -translate-y-1/2 rotate-0 text-2xl transition-all duration-100 ease-in-out group-hover:-ml-8 group-hover:-rotate-45">
```

Reply:
I`ll fix the size of the monster and the monster will appear on the other side, because the button is the similar color.

![back-to-top-btn-2](https://user-images.githubusercontent.com/41576384/216759694-c40675a7-d7f9-4121-9394-1e9da8548717.png)

Reply:
Looks great